source/includes/options-mongod.yaml

```diff
@@ -1140,6 +1140,20 @@ description: |
   Configures this {{program}} instance as a shard in a
   sharded cluster. The default port for these instances is
   ``27018``.
+
+  .. important::
+
+     .. versionchanged:: 3.6
+
+     You must deploy shard servers as a replica set with the
```

Comment:
shard servers -> shards

(since the individual servers are components of the repl set)